repos/rust-lang/cargo.toml

```diff
@@ -2,13 +2,14 @@ org = "rust-lang"
 name = "cargo"
 description = "The Rust package manager"
 homepage = "https://doc.rust-lang.org/cargo"
-bots = ["bors", "rustbot", "rfcbot", "renovate"]
+bots = ["rustbot", "rfcbot", "renovate"]
 
 [access.teams]
 cargo = "write"
 
 [[branch-protections]]
 pattern = "master"
+ci-checks = ["conclusion"]
```

Comment:
ci-checks will be needed on the `rust-1.*` branch protection, too. We use those for beta backports.

Reply:
Or...hm.. Merge queues don't support `*` patterns. That's...unfortunate.

Reply:
If it's needed only for cargo, I would just configure it manually. TIL about the `*` limitation, that's annoying.

Reply:
Just checking, I think it would be good to also add ci-checks to the beta branch to ensure that nothing is merged with failures.

Reply:
Changed.